web/src/styles/scss/header.scss

```diff
@@ -1,21 +1,115 @@
 @import '@cmsgov/design-system-support/src/settings/index';
 
-/* we may be able to just target the header element without a class in future iterations. */
-.header__slim {
-  background-color: $color-primary-alt-darkest;
-  padding: $spacer-1 0;
-  .site-title {
-    display: block;
-    margin: $spacer-1 0;
-    text-decoration: none;
-    font-weight: bold;
+header {
+  background-color: $color-primary-darkest;
+  padding: $spacer-2 0;
+  letter-spacing: 0.6px;
```

Comment:
@austinhernandez I kept the original letter spacing here since we haven't moved to the new font family yet. If you'd rather I nix it, let me know!